docs/shared/guides/angular-nx-version-matrix.md

```diff
@@ -0,0 +1,18 @@
+# Nx and Angular Version Compatibility Matrix
+
+Below is a reference table that matches versions of Angular to the version of Nx that is compatible with it. The table shows the version of Angular, the recommended version of Nx to use and the range of Nx versions that support the version of Angular.
+
+We provide a recommended version, and it is usually the latest minor version of Nx in the range provided because there will have been bug fixes added since the first release in the range.
```

Comment:
>there will have been bug fixes
I'm not sure to understand what you meant here.

Reply:
so, say the earliest version we support Angular 14 is Nx 14.1 but we're currently on Nx 14.5, then we'd recommend Nx 14.5 to use Angular 14 with as it may contain bug fixes for issues that we were made aware of between Nx 14.1 and Nx 14.5